.tool-versions

```diff
@@ -0,0 +1 @@
+java corretto-21.0.5.11.1
```

Comment:
We were using Java 11 with `actions/setup-java`. Here we're using Java 21. Have we tested with this version?

Reply:
The deployment process here is a bit mysterious.  I have only tested the CI workflow.  I'll set the version to 11 to keep this as an atomic change.